Replicator/Replicator.cc

```diff
@@ -817,6 +820,11 @@ namespace litecore { namespace repl {
             MessageIn *response = progress.reply;
 
             if (response->isError()) {
+                if (response->getError().message == "No handler for BLIP request") {
```

Comment:
Is it possible to check error domain and code instead of message?

Reply:
I am worried that the error code (404) is quite general and not specific to this error. All I could find internally about it was that 404 corresponds to "Not Found" (for anything). And I spoke to Ben about it, he said there could be other errors which return a 404.

Reply:
The same logic applies for your addition of "remote error" into the stopping error list.  This applies to any error that happened on the Sync Gateway side, basically, and not just in getCollections.